cliff.toml

```diff
@@ -45,23 +45,44 @@ postprocessors = [
 # parse the commits based on https://www.conventionalcommits.org
 conventional_commits = true
 # filter out the commits that are not conventional
-filter_unconventional = true
+filter_unconventional = false
 # process each line of a commit as an individual commit
 split_commits = false
 # regex for parsing and grouping commits
 commit_parsers = [
-  { message = "^feat", group = "Features" },
-  { message = "^fix", group = "Bug Fixes" },
-  { message = "^doc", group = "Documentation" },
-  { message = "^perf", group = "Performance" },
-  { message = "^refactor", group = "Refactor" },
-  { message = "^style", group = "Styling" },
-  { message = "^test", group = "Testing" },
-  { message = "^chore\\(release\\): prepare for", skip = true },
-  { message = "^chore\\(deps\\)", group = "Dependencies" },
-  { message = "^chore", group = "Miscellaneous Tasks" },
-  { message = "^ci", group = "CI/CD" },
-  { body = ".*security", group = "Security" },
+  # Conventional commits with scope
+  { message = "^feat(\\(.*\\))?:", group = "Features" },
+  { message = "^fix(\\(.*\\))?:", group = "Bug Fixes" },
+  { message = "^docs(\\(.*\\))?:", group = "Documentation" },
+  { message = "^doc(\\(.*\\))?:", group = "Documentation" },
+  { message = "^perf(\\(.*\\))?:", group = "Performance" },
+  { message = "^refactor(\\(.*\\))?:", group = "Refactor" },
+  { message = "^style(\\(.*\\))?:", group = "Styling" },
+  { message = "^test(\\(.*\\))?:", group = "Testing" },
+  { message = "^chore\\(release\\):", skip = true },
+  { message = "^chore\\(deps\\)(\\(.*\\))?:", group = "Dependencies" },
+  { message = "^chore(\\(.*\\))?:", group = "Miscellaneous Tasks" },
+  { message = "^ci(\\(.*\\))?:", group = "CI/CD" },
+  
+  # Simple commits without scope
+  { message = "^feat\\s+", group = "Features" },
+  { message = "^fix\\s+", group = "Bug Fixes" },
+  { message = "^doc\\s+", group = "Documentation" },
+  { message = "^test\\s+", group = "Testing" },
+  
+  # Ticket prefixed commits
+  { message = "^MTG-\\d+\\s+.*(fix|fixed)", group = "Bug Fixes" },
+  { message = "^MTG-\\d+\\s+.*(feat|feature)", group = "Features" },
+  { message = "^MTG-\\d+\\s+.*(doc|docs)", group = "Documentation" },
+  { message = "^MTG-\\d+\\s+.*(test)", group = "Testing" },
+  { message = "^MTG-\\d+\\s+", group = "Tasks" },
+  
+  # Merge commits
+  { message = "^Merge pull request .*/fix/", group = "Bug Fixes" },
+  { message = "^Merge pull request .*/feat/", group = "Features" },
+  { message = "^Merge pull request .*/doc/", group = "Documentation" },
+  { message = "^Merge pull request .*/test/", group = "Testing" },
+  { message = "^Merge", group = "Merges" },
```

Comment:
i think these don't need to be accounted for. ticket prefixed commits are probably fine, but merges will just duplicate stuff.